### PR TITLE
Update StackSplit for v3.1 regarding docs and comments

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
This PR includes changes and updates of StackSplit for the v3.1 release regarding the documentation and comments.

**Changes**
- [x] Improve coding style
- [x] Remove un-needed variable definitions
- [x] Remove un-needed brackets
- [x] Fix typos

There is a separate PR #12 which handels changes regarding code and R2023a to replace deprecated or not recommanded MATLAB functions:
   - `resizem()` -> `georesize()` / `imresize()`: `SS_check_input.m` &nbsp;&nbsp;&nbsp; will be removed
   - `datenum()` -> `datetime()`: `SS_prep_SIMW.m`, `SS_stack_Esurf.m` &nbsp;&nbsp;&nbsp; not recommanded
   - `datestr()` -> `datetime()`: `SS_read_SLresults.m`, `SS_saveresults.m` &nbsp;&nbsp;&nbsp; not recommanded
   - `now` -> `covertTo()`: `SS_saveresults.m` &nbsp;&nbsp;&nbsp; not recommanded

**Testing**
|MATLAB|Windows|Linux|
|---|---|---|
|R2018a|no license|:white_check_mark:|
|R2022a|not tested|:white_check_mark:|
|R2022b|:white_check_mark:|not tested|
|R2023a (pre-release)|:white_check_mark:|no license|

### CODE REVIEW

**Comment by yvonnefroehlich** [comment]
Maybe it makes sense to split these changes into two PRs, as this PR is getting longer as expected? One for the more formal changes (this PR) and one for the more code related changes (a second PR)?

**Comment by yvonnefroehlich** [comment]
@michaelgrund are you fine with removing trailing white spaces from all StackSplit functions?

File: StackSplit/SS_splitdiagnosticSetHeader.m [hunk 1]
```diff
@@ -44,12 +44,12 @@ function SS_splitdiagnosticSetHeader(axH, phiRC, dtRC, phiSC, dtSC, phiEV, dtEV,
...
-str12 = sprintf(['%4.0f<%4.0f\\circ <%4.0f']   ,phiSC);
-str22 = sprintf('%3.1f<%3.1fs<%3.1f'           ,dtSC);
-str13 = sprintf(['%4.0f<%4.0f\\circ <%4.0f']   ,phiEV);
-str23 = sprintf('%3.1f<%3.1fs<%3.1f'           ,dtEV);
+str11 = sprintf('%4.0f <%4.0f\\circ <%4.0f', phiRC);
+str21 = sprintf('%3.1f < %3.1fs < %3.1f', dtRC);
+str12 = sprintf('%4.0f <%4.0f\\circ <%4.0f', phiSC);
+str22 = sprintf('%3.1f < %3.1fs < %3.1f', dtSC);
+str13 = sprintf('%4.0f <%4.0f\\circ <%4.0f', phiEV);
+str23 = sprintf('%3.1f < %3.1fs < %3.1f', dtEV);
```
Comment (michaelgrund) [inline StackSplit/SS_splitdiagnosticSetHeader.m]: Have you checked that the header visually still looks fine when removing the whitespaces here?

Comment (yvonnefroehlich) [reply StackSplit/SS_splitdiagnosticSetHeader.m]: At the moment I am unfortunately not completely sure which white spaces you mean. The white spaces I removed are all outside of the text string. The added white spaces should add consistently white spaces before and after the `<` signs.

_Diagnostic plot with main branch_:
StackSplit_Multi_result_SIMW_main

_Diagnostic plot with changes in this PR_:
StackSplit_Multi_result_SIMW_PR11

Comment (michaelgrund) [reply StackSplit/SS_splitdiagnosticSetHeader.m]: Ok fine!

**Comment by michaelgrund** [comment]
Anything else you wanna fix in this PR @yvonnefroehlich ?

**Comment by yvonnefroehlich** [comment]
> Anything else you wanna fix in this PR @yvonnefroehlich ?

I think, these are enough changes for one PR... I am sorry, I didn't expect this! Despite this, no guarantee that I found every typo!

**Comment by yvonnefroehlich** [comment]
> @michaelgrund are you fine with removing trailing white spaces from all StackSplit functions?

But one thing: Do you want me to remove the trailing white spaces @michaelgrund? Probably this comment was overlooked, as GitHub already hides some items...

**Comment by michaelgrund** [comment]
> > @michaelgrund are you fine with removing trailing white spaces from all StackSplit functions?
> 
> But one thing: Do you want me to remove the trailing white spaces @michaelgrund? Probably this comment was overlooked, as GitHub already hides some items...

I'm fine with that!

**Comment by yvonnefroehlich** [comment]
> > > @michaelgrund are you fine with removing trailing white spaces from all StackSplit functions?
> > 
> > 
> > But one thing: Do you want me to remove the trailing white spaces @michaelgrund? Probably this comment was overlooked, as GitHub already hides some items...
> 
> I'm fine with that!

These changes are done in the commits https://github.com/michaelgrund/stacksplit/pull/11/commits/00d22435e70a3895c45dd9a18b17abe3b03ff771, https://github.com/michaelgrund/stacksplit/pull/11/commits/5b54c85f68340d81133a6a5f080236cc670369a0, and https://github.com/michaelgrund/stacksplit/pull/11/commits/80bc1453e7aa97509a1915be03e2a20830889f5a.

File: StackSplit/Doc/SL2SS_changelog.txt [hunk 2]
```diff
@@ -9,5 +9,5 @@ splitdiagnosticplot.m    adjustments to save new outputs temporary
 saveresult.m             adjustments to save new outputs finally
 database editResults.m   adjustments to avoid database conflicts
 seisfigbuttons.m         adjustments to avoid database conflicts
-getFileAndEQseconds.m    fixed start time extraction by SplitLab (v3.0, see Fr�hlich et al. 202x)
+getFileAndEQseconds.m    fixed start time extraction by SplitLab (v3.0, see Fr�hlich et al., 2022)
```
Comment (yvonnefroehlich) [inline StackSplit/Doc/SL2SS_changelog.txt]:
```suggestion
getFileAndEQseconds.m    fixed start time extraction by SplitLab (v3.0, see Fröhlich et al., 2022)
```

**Review by michaelgrund — APPROVED** [review]

**Pull request merged**